x/ccv/types/ccv.go

```diff
@@ -61,9 +63,6 @@ func (vdt SlashPacketData) ValidateBasic() error {
 	if len(vdt.Validator.Address) == 0 || vdt.Validator.Power == 0 {
 		return sdkerrors.Wrap(ErrInvalidPacketData, "validator fields cannot be empty")
 	}
-	if vdt.ValsetUpdateId == 0 {
```

Comment:
For clarity, this is possible since the consumer is setting the default value of `HeightValsetUpdateID` to `0` during `InitGenesis`, i.e.,
```golang
// set default value for valset update ID
k.SetHeightValsetUpdateID(ctx, uint64(ctx.BlockHeight()), uint64(0))
```
and is using this value to send `SlashPackets` for infractions that happen before receiving the first `VSCPacket` when the value of `HeightValsetUpdateID` is updated, i.e.,
```golang
k.SetHeightValsetUpdateID(ctx, uint64(ctx.BlockHeight())+1, newChanges.ValsetUpdateId)
```